src/OSPSuite.UI/Services/IImageListRetriever.cs

```diff
@@ -6,12 +6,12 @@ namespace OSPSuite.UI.Services
 {
    public interface IImageListRetriever : IStartable
    {
-      ImageCollection AllImages16x16 { get; }
-      ImageCollection AllImages24x24 { get; }
-      ImageCollection AllImages32x32 { get; }
-      ImageCollection AllImagesForContextMenu { get; }
-      ImageCollection AllImagesForTreeView { get; }
-      ImageCollection AllImagesForTabs { get; }
+      SvgImageCollection AllImages16x16 { get; }
+      SvgImageCollection AllImages24x24 { get; }
+      SvgImageCollection AllImages32x32 { get; }
+      SvgImageCollection AllImagesForContextMenu { get; }
+      SvgImageCollection AllImagesForTreeView { get; }
+      SvgImageCollection AllImagesForTabs { get; }
```

Comment:
I wonder where is the size 48x48, which is available in our options?
![grafik](https://user-images.githubusercontent.com/25061876/160088152-25f11ebe-8235-4842-bc66-9db041e171df.png)

Reply:
This size thing is a bit strange with avg. They scale continuously so forcing a size is maybe not even what we need.

Reply:
ok I see. This is defined just not exposed directly. 
```

      public void Start()
      {
         if (_initialized) return;
         _allImages16x16 = _mapper.MapFrom(ApplicationIcons.All(), IconSizes.Size16x16);
         _allImages24x24 = _mapper.MapFrom(ApplicationIcons.All(), IconSizes.Size24x24);
         _allImages32x32 = _mapper.MapFrom(ApplicationIcons.All(), IconSizes.Size32x32);
         _allImages48x48 = _mapper.MapFrom(ApplicationIcons.All(), IconSizes.Size48x48);
         _initialized = true;
      }

      public SvgImageCollection AllImagesForTreeView => allImageForSize(_userSettings.IconSizeTreeView);

      public SvgImageCollection AllImagesForContextMenu => allImageForSize(_userSettings.IconSizeContextMenu);

      public SvgImageCollection AllImagesForTabs => allImageForSize(_userSettings.IconSizeTab);
```